pkg/transport/manager.go

```diff
@@ -305,37 +305,11 @@ func (tm *Manager) SaveTransport(ctx context.Context, remote cipher.PubKey, tpTy
 	for {
 		mTp, err := tm.saveTransport(remote, tpType, label)
```

Comment:
Should we pass context here that we accept in `SaveTransport` instead of creating a new context in `saveTransport`?

Reply:
Yes, your right, I had forgotten context in `SaveTransport`.